src/config/dynamic-theme-fixes.config

```diff
@@ -11037,6 +11037,21 @@ INVERT
 
 ================================
 
+rain.thecomicseries.com
+
+CSS
+body {
```

Comment:
Just to add a note, I considered using this instead:

```CSS
body {
    background-color: inherit !important;
    background-image: none !important;
}
```

This would use the `html` tag background color that Dark Reader adds. However, this doesn't look terribly good. Notably, this results in the loss of the intended center column in light mode.

Reply:
You could simply add `background-color: var(--darkreader-neutral-background) !important;` to `html` tag, such that you would keep consistency between body and html